{"level": "DEBUG", "message": "debug message", "timestamp": "2025-03-18T20:25:41.665020+00:00", "logger": "__main__", "module": "main", "function": "main", "line": 37, "thread_name": "MainThread"}
{"level": "INFO", "message": "info message", "timestamp": "2025-03-18T20:25:41.665067+00:00", "logger": "__main__", "module": "main", "function": "main", "line": 38, "thread_name": "MainThread"}
{"level": "WARNING", "message": "warning message", "timestamp": "2025-03-18T20:25:41.665089+00:00", "logger": "__main__", "module": "main", "function": "main", "line": 39, "thread_name": "MainThread"}
{"level": "ERROR", "message": "error message", "timestamp": "2025-03-18T20:25:41.665101+00:00", "logger": "__main__", "module": "main", "function": "main", "line": 40, "thread_name": "MainThread"}
{"level": "CRITICAL", "message": "critical message", "timestamp": "2025-03-18T20:25:41.665109+00:00", "logger": "__main__", "module": "main", "function": "main", "line": 41, "thread_name": "MainThread"}
{"level": "ERROR", "message": "exception message\nTraceback (most recent call last):\n  File \"/Users/pfury/Documents/projects/flashcards_ai_obsidian_pro/app/main.py\", line 44, in main\n    1 / 0\n    ~~^~~\nZeroDivisionError: division by zero", "timestamp": "2025-03-18T20:25:41.665124+00:00", "logger": "__main__", "module": "main", "function": "main", "line": 46, "thread_name": "MainThread"}
{"level": "DEBUG", "message": "debug message", "timestamp": "2025-03-18T20:29:24.207427+00:00", "logger": "__main__", "module": "main", "function": "main", "line": 39, "thread_name": "MainThread"}
{"level": "INFO", "message": "info message", "timestamp": "2025-03-18T20:29:24.207468+00:00", "logger": "__main__", "module": "main", "function": "main", "line": 40, "thread_name": "MainThread"}
{"level": "WARNING", "message": "warning message", "timestamp": "2025-03-18T20:29:24.207493+00:00", "logger": "__main__", "module": "main", "function": "main", "line": 41, "thread_name": "MainThread"}
{"level": "ERROR", "message": "error message", "timestamp": "2025-03-18T20:29:24.207509+00:00", "logger": "__main__", "module": "main", "function": "main", "line": 42, "thread_name": "MainThread"}
{"level": "CRITICAL", "message": "critical message", "timestamp": "2025-03-18T20:29:24.207517+00:00", "logger": "__main__", "module": "main", "function": "main", "line": 43, "thread_name": "MainThread"}
{"level": "ERROR", "message": "exception message\nTraceback (most recent call last):\n  File \"/Users/pfury/Documents/projects/flashcards_ai_obsidian_pro/app/main.py\", line 46, in main\n    1 / 0\n    ~~^~~\nZeroDivisionError: division by zero", "timestamp": "2025-03-18T20:29:24.207528+00:00", "logger": "__main__", "module": "main", "function": "main", "line": 48, "thread_name": "MainThread"}
